src/main/kotlin/no/nav/faktureringskomponenten/domain/models/FakturaLinje.kt

```diff
@@ -24,6 +24,9 @@ class FakturaLinje(
     @Column(name = "belop", nullable = false)
     val belop: BigDecimal = BigDecimal(0),
 
+    @Column(name = "antall", nullable = false)
+    val antall: BigDecimal = BigDecimal(0),
```

Comment:
Kunne vi fått bedre navn enn antall? Føler det er veldig vanskleig å vite hva det beskriver. "antallEnhetspriser" er egentlig mer beskrivende ord. Det tok meg lang tid å skjønne hva `AntallBeregner` var for noe for eksempel, selv om jeg var fullstendig klar over hva antall representerte i hodet mitt.

Reply:
Jeg er helt enig, men det er ikke jeg som bestemmer hva fagfolk velger å kalle det. Når vi prater om antall i fakturalinje så forstår de

Reply:
tja, dette er jo internt i databasen vår og gjør det enklere for oss å forstå. Vi eksponerer ikke "antallEnhetspriser" ut noen sted.

Reply:
Har du forslag?

Reply:
`antallEnhetspriser` er jo forslaget, men mulig det er også dårlig/uklart.

Reply:
![image](https://user-images.githubusercontent.com/6776092/225606847-ec1b59d0-afb9-4a46-8938-f6fa73f29265.png)
I kontekset gir antall mening. Vi kan også evt. bare endre navn på Entity'en uten å endre kolonnenavn 

Hva tenker du @janitamarielle, skal vi endre navn?

Reply:
Jeg syns antall er klart så lenge man vet det handler om faktura-linje. Er jo noe av det første man blir introdusert med når man begynner å jobbe med faktureringskomponenten.

Reply:
Enig i det, lar den være for øyeblikket så kan vi evt. ta denne opp igjen senere

Reply:
Enig i det, lar den være for øyeblikket så kan vi evt. ta denne opp igjen senere